From ec986ff5fbb90e38b7766d45c80ad0e9bd83959f Mon Sep 17 00:00:00 2001
From: Randy Li <randy.li@rock-chips.com>
Date: Thu, 21 Jun 2018 10:22:17 +0800
Subject: [PATCH] ARM64: dts: rockchip: fix rk3399 device aliases

Enable the alias for ethernet in rk3399 and order the items
in alphabetical order.

Change-Id: I7862593ad99028eba75102fb684987cec8762d1c
Signed-off-by: Randy Li <randy.li@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399.dtsi | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399.dtsi b/arch/arm64/boot/dts/rockchip/rk3399.dtsi
index d026527ee879..7e2f45515622 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399.dtsi
@@ -61,6 +61,9 @@
 	#size-cells = <2>;
 
 	aliases {
+		dsi0 = &dsi;
+		dsi1 = &dsi1;
+		ethernet0 = &gmac;
 		i2c0 = &i2c0;
 		i2c1 = &i2c1;
 		i2c2 = &i2c2;
@@ -75,8 +78,6 @@
 		serial2 = &uart2;
 		serial3 = &uart3;
 		serial4 = &uart4;
-		dsi0 = &dsi;
-		dsi1 = &dsi1;
 	};
 
 	cpus {
-- 
2.35.3

